docs/source/markdown/podman-image-sign.1.md

```diff
@@ -36,6 +36,10 @@ Store the signatures in the specified directory.  Default: /var/lib/containers/s
 
 Override the default identity of the signature.
 
+#### **--authfile**=*file*
+
+Override the default auth file of the signature.
```

Comment:
It's not really the auth file **of** the signature, but the file used to look up credentials when contacting registries.

For the sake of consistency with all other commands with --authfile, please use the same description (e.g., https://github.com/containers/podman/blob/main/docs/source/markdown/podman-pull.1.md#--authfilepath).